tests/run-make/branch-protection-check/Makefile

```diff
@@ -0,0 +1,12 @@
+# Check for GNU Property Note
+
+include ../tools.mk
+
+# How to run this
+#RUSTFLAGS_NOT_BOOTSTRAP=-Zcf-protection=branch python3 x.py test --target x86_64-unknown-linux-gnu --stage 1  tests/run-make/branch-protection-check/
```

Comment:
Why does this require setting RUSTFLAGS_NOT_BOOTSTRAP? Does that mean it's untested in CI?

Reply:
I needed the flag when building the compiler locally, but did not need it to run the test (and removed it from the comment). Was there something else we should be doing here?

Reply:
I am not sure why you needed the flag when building locally, but no, the test looks fine as-is :)